backend/app/controllers/spree/admin/general_settings_controller.rb

```diff
@@ -27,7 +27,7 @@ def clear_cache
       end
 
       def render(*args)
-        @preferences_currency |= [:allow_currency_change, :show_currency_selector]
+        @preferences_currency |= [:allow_currency_change]
```

Comment:
@KacperMekarski can we rename it to `show_store_selector` ? To better reflect the current status?

Reply:
@damianlegawiec done.